content/blog/2024/css-demographics.md

```diff
@@ -0,0 +1,76 @@
+---
+title: What do survey demographics tell us?
+sub: Are we measuring what we meant to measure?
+date: 2024-10-31
+author: miriam
+tags:
+  - CSS
+summary: |
+  There's been a lot of interest
+  in the results of the annual
+  State of CSS survey,
+  but are we asking all the right questions?
+---
+
+Sacha Greif's annual
+[State of CSS survey](https://2024.stateofcss.com/en-US/)
+wrapped up recently,
+and the results are online.
+
+I've seen a number of people
+commenting on the
+[demographics data](https://2024.stateofcss.com/en-US/demographics/),
+asking what it says about our industry.
+Those are great questions to ask.
+For example:
+men in the US seem to make significantly more money
+than the rest of the field.
+
+But I haven't seen analysis asking
+if the survey demographics actually match the industry.
+And for this,
+I would include a number of demographic questions
+from the 'CSS Usage' section:
+like the [industry sector](https://2024.stateofcss.com/en-US/usage/#industry_sector),
+[CSS purpose](https://2024.stateofcss.com/en-US/usage/#usage_type),
+and the
+[balance of time respondents spend writing CSS (and HTML) vs JS](https://2024.stateofcss.com/en-US/usage/#css_js_balance).
+Do we know how representative this is?
+
+- Are 80% of global CSS authors actually white?
+- Are 90% of CSS authors men?
+- Do the majority of CSS authors spend more time writing JS?
+- Is CSS mostly used for web apps?
+- Does the industry skew young to the same degree as the survey?
+
+If you don't know [Kevin Powell's work teaching CSS](https://www.kevinpowell.co/),
+I highly recommend it.
+But as you might expect,
+his audience is weighted towards students
+and people starting out in the field.
+If he's a [primary source for respondents](https://2024.stateofcss.com/en-US/demographics/#source),
+that's likely to impact the demographics.
+
+And it's great to get that perspective!
+But it still requires us to ask:
+are we looking at data about CSS authors generally,
+or are we looking at a small slice of the industry?
+And is that slice more broadly representative?
+
+In many cases,
+polls use demographics data to
+_adjust for response bias_ --
+but that requires an understanding of how the demographics
+interact with other data points.
+How would it change our interpretation of the survey more generally
+if we knew the answers to those questions?
+
+- Do most CSS authors test their designs only on desktop,
+  or is that specific to people who write CSS on the side?
+- Is Tailwind broadly popular,
+  or within certain demographics?
+- What other questions might rise up from this analysis?
+
+I don't have answers here.
+I'm just not sure what I'm looking at,
+or how it should be read.
```

Comment:
Can we end with a CTA? Maybe asking what we missed or a poll or something to see if our readers filled out that survey (I didn't). 

I am ALWAYS critiquing surveys for the questions they didn't ask, or for the ways in which I would have worded something differently. I'm not saying anything about this survey especially since I didn't take it, but I tend to have an issue with many surveys I see.